gazebo_ros/src/gazebo_ros_init.cpp

```diff
@@ -176,6 +176,10 @@ void GazeboRosInit::Load(int argc, char ** argv)
     rclcpp::ParameterValue(GazeboRosInitPrivate::DEFAULT_PUBLISH_FREQUENCY));
   impl_->throttler_ = Throttler(rate_param.get<double>());
 
+  // PerformanceMetrics parameter
+  impl_->ros_node_->declare_parameter<bool>("enable_performance_metrics", true);
```

Comment:
I don't think the performance metrics feature is really documented anywhere. It might be nice to pass a [ParameterDescriptor](https://github.com/ros2/rclcpp/blob/0034929eef9f6626530d8207030b791e121a5415/rclcpp/include/rclcpp/node.hpp#L434) here with the `description` field set to document what this parameter does (e.g. "if set to true, performance metrics are published to topic 'performance_metrics'").

Reply:
Done